shadowsocks-csharp/View/StatisticsStrategyConfigurationForm.cs

```diff
@@ -31,11 +31,31 @@ public StatisticsStrategyConfigurationForm(ShadowsocksController controller)
             _pingSeries = StatisticsChart.Series["Ping"];
             _controller = controller;
             _controller.ConfigChanged += (sender, args) => LoadConfiguration();
+            UpdateTexts();
             LoadConfiguration();
             Load += (sender, args) => InitData();
-
         }
 
+        private void UpdateTexts()
+        {
+            Control[] controls = new Control[]
+            {
+                StatisticsEnabledCheckBox,PingCheckBox,label6,
```

Comment:
You can give labels a meaningful name rather than the existing auto generated name. Such as: label6 --> labelPackagesEverytime